### PR TITLE
shrink down final container

### PR DESCRIPTION
**Pull request by kpirnie**
## Context 

Final container size down to 19.6M

## Test instructions

1. Just rebuild your local container and run it

## Checklist before requesting a review

* [X] I have performed a self-review of my code
* [X] I've not introduced breaking changes.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,7 +13,22 @@ RUN go mod download
 # Copy the source code from the current directory to the Working Directory inside the container
 COPY . .
 
-RUN go test ./... \
-  && go build -ldflags='-s -w' -o main .
+RUN go build -ldflags='-s -w' -o main .
```
Comment (sonroyaalmerol) [inline Dockerfile:16]: Don't remove the test execution on build stage

File: Dockerfile [hunk 2]
```diff
@@ -13,7 +13,22 @@ RUN go mod download
...
 
-ENTRYPOINT ["/app/main"]
+# End from the latest alpine image
+FROM alpine:latest
```
Comment (sonroyaalmerol) [inline Dockerfile:19]: Add `# hadolint ignore=DL3007` above this line to ignore the linter warning.

File: Dockerfile [hunk 3]
```diff
@@ -13,7 +13,22 @@ RUN go mod download
...
+
+# add bash and timezone data
+RUN \
```
Comment (sonroyaalmerol) [inline Dockerfile:22]: Add `# hadolint ignore=DL3008` above this line to ignore another linter warning.

File: Dockerfile [hunk 4]
```diff
@@ -13,7 +13,22 @@ RUN go mod download
...
+  apk update && \
+  apk add bash && \
```
Comment (sonroyaalmerol) [inline Dockerfile:24]: I see no particular reason why the proxy would need bash as a dependency. The same thing can be said with htop.

**Comment by sonroyaalmerol** [comment]
Hi! See my reviews above and try to make sure both Linting checks are successful before I merge this. You can safely ignore the Static Code analysis check. Thanks!

File: Dockerfile (hunk 1 above)
Comment (kpirnie) [reply Dockerfile:16]: I remove this because it fails to build requiring Redis

File: Dockerfile (hunk 4 above)
Comment (kpirnie) [reply Dockerfile:24]: yeah, i added those in mine to watch the processes.

**Comment by kpirnie** [comment]
> Hi! See my reviews above and try to make sure both Linting checks are successful before I merge this. You can safely ignore the Static Code analysis check. Thanks!

hate to ask, but how do I check the linting? (I've never done a pull request...)

**Comment by sonroyaalmerol** [comment]
No problem! You can see the checks which is done automatically on every commit. The list of checks will probably be below this comment (usually below the latest comment). You can view the details (including the errors) of each checks there as well.

**Comment by sonroyaalmerol** [comment]
I've went ahead and done some commit myself to resolve the issues. I wanted to include it in the next release. Thanks for your work!

**Pull request merged**